### PR TITLE
Read new repositories

### PR DESCRIPTION
**Pull request by johnoliver**
Modify code to first check the new `openjdk<x>-binaries` repos to look for releases, if none can be found in those locations default to the old repos.

##### Checklist

- [x] `npm test` passes
- [x] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/js/0-global.js [hunk 1]
```diff
@@ -138,6 +143,47 @@ function detectOS() {
...
+  }
+  return response
+}
+
+// load latest_nightly.json/nightly.json/releases.json/latest_release.json files
+// This will first try to load from openjdk<X>-binaries repos and if that fails
+// try openjdk<X>-release, i.e will try the following:
+
+// https://github.com/AdoptOpenJDK/openjdk10-binaries/blob/master/latest_release.json
+// https://github.com/AdoptOpenJDK/openjdk10-releases/blob/master/latest_release.json
```
Comment (karianna) [inline src/js/0-global.js]: This is only for 'release' level builds or 'nightly' level builds?

Comment (johnoliver) [reply src/js/0-global.js]: No, those are just examples, in fact at the moment I have only pushed nightlys, so only nightlys will be found, but either should work

File: src/js/0-global.js [hunk 2]
```diff
@@ -222,13 +255,21 @@ function setTickLink() {
...
 }
 
-function setUrlQuery(name, newValue) {
-  if(window.location.search.indexOf(name) >= 0) {
-    var currentValue = getQueryByName(name);
-    window.location.search = window.location.search.replace(currentValue, newValue);
-  }
-  else {
-    window.location.search += (name + '=' + newValue);
+function setUrlQuery() {
```
Comment (karianna) [inline src/js/0-global.js]: Might be nice to show a sample in a comment here

File: src/js/0-global.js [hunk 3]
```diff
@@ -265,6 +306,8 @@ function persistUrlQuery() {
   });
 }
 
+const versionMatcher=/(openjdk\d+)-([a-zA-Z0-9]+)/;
```
Comment (karianna) [inline src/js/0-global.js]: Declare at the top if it's global?

File: src/js/archive.js [hunk 4]
```diff
@@ -10,47 +10,36 @@ function onArchiveLoad() {
...
-          buildArchiveHTML(releasesJson, jckJSON);
-        });
-      } else { // if there are no releases (beyond the latest one)...
-        // report an error, remove the loading dots
-        loading.innerHTML = '';
-        errorContainer.innerHTML = '<p>There are no archived releases yet! See the <a href=\'./releases.html?variant=' + variant + '\'>Latest release</a> page.</p>';
+      if (response !== null) {
+        buildArchiveHTML(response);
+      } else {
+        if(oldRepo) {
```
Comment (karianna) [inline src/js/archive.js]: Not sure why this logic is wrapped in oldRepo?

Comment (johnoliver) [reply src/js/archive.js]: The sequence is we check the new repo, if there are no files then check the old one, this then says if we find nothing from the old, display an error

**Pull request merged**